### ISSUE TITLE
Bx.in.th Incorrect 2FA Code when create order

### ISSUE THREAD

**Issue by kmhcool** [ccxt/ccxt#2103]
- OS: Ubuntu 16.04.3 LTS
- Programming Language version: Python 3.5.2
- CCXT version: 1.10.1266
- Exchange: Bx.in.th
- Method: create_limit_buy_order, create_limit_buy_order

I use this code

```
trading_client = ccxt.bxinth({'apiKey':api_key,'secret':api_secret,'verbose': True,'enableRateLimit': True,'nonce': lambda: ccxt.Exchange.milliseconds()})
print(trading_client.create_limit_sell_order(symbol,amount,float(price)+price_interval,{'twofa':totp.now()}))
```

 I want to use 2fa in my account and use the api at the same time. Then, I got this error

>   File "<string>", line 206, in <module>
>   File "/usr/local/lib/python3.5/dist-packages/ccxt/base/exchange.py", line 1171, in create_limit_sell_order
>     return self.create_order(symbol, 'limit', 'sell', *args)
>   File "/usr/local/lib/python3.5/dist-packages/ccxt/bxinth.py", line 207, in create_order
>     }, params))
>   File "/usr/local/lib/python3.5/dist-packages/ccxt/bxinth.py", line 286, in request
>     raise ExchangeError(self.id + ' ' + self.json(response))
> ccxt.base.errors.ExchangeError: bxinth {"order_id":0,"error":"Incorrect 2FA Code","success":false}

and as I try to execute 
`print(trading_client.create_limit_sell_order(symbol,amount,float(price)+price_interval,{'twofa':totp.now()}))`
in the interpreter, it works normally without any problem
but as I try with python3 command, it always got stuck with this error.

edit:

Look like my editor screw my head for a bit and doesn't add verbose to my code when I first post this, anyway the output for 2fa is 

> Request: POST https://bx.in.th/api/order/?pairing=X&amount=X&type=sell&rate=XXX.X&signature=XXX&twofa=XXXXXX {'Accept-Encoding
> ': 'gzip, deflate', 'User-Agent': 'python-requests/2.18.4', 'Content-Type': 'application/x-www-form-urlencoded'} nonce=1519930874853&pairing=X&amount=X&type=sell&rate=XXX.X&signature=XXX&twofa=XXXXXX&key=X
> 
> Response: POST https://bx.in.th/api/order/?pairing=X&amount=X&type=sell&rate=XXX.X&twofa=XXXXXX 200 {'X-Content-Type-Options': 'nosniff', 'Expect-CT': 'max-age=604800, report-uri="https://report-uri.cloudflare.com/cdn-cgi/beacon/expect-ct"', 'Content-Length': '79', 'Set-Cookie': '__cfduid=d4ebfcaa9cddde382cb90f0afe11f9fff1519930874; expires=Fri, 01-Mar-19 19:01:14 GMT; path=/; domain=.bx.in.th; HttpOnly; Secure', 'Date': 'Thu, 01 Mar 2018 19:01:15 GMT', 'Server': 'cloudflare', 'CF-RAY': '3f4ddeffdb0f79ed-SEA', 'Vary': 'Accept-Encoding', 'Content-Encoding': 'gzip', 'Strict-Transport-Security': 'max-age=0', 'Expires': 'Fri, 02 Mar 2018 19:01:15 GMT', 'Connection': 'keep-alive', 'Content-Type': 'text/javascript', 'Cache-Control': 'max-age=86400'} {"success":false,"order_id":0,"error":"Incorrect
>  2FA Code"}

Looking forward to your response and thank you so much.

**Comment by kroitor** [comment]
@kmhcool hi! Does it also throw the same error if you fill the 2FA-code by hand? In other words, have you verified that the code itself corresponds to the code shown on the linked device?

**Comment by kmhcool** [comment]
I think I got it now. The problem is I try to use the 2FA code that I used before. It's an One Time Password and I forgot that. So, the old 2FA code that got used won't work (of course it won't). I think we can closed this issue now. Thank you so much.

By the way, the story is that I don't provide enough code for this problem myself too

`tmp_fetch_open_orders = trading_client.fetchOpenOrders(symbol,params={'twofa':totp.now()})
print(str(trading_client.create_limit_sell_order(symbol,amount,float(price)+price_interval,{'twofa':totp.now()})))
`

something like this, and I used the same code and got that error.
So my solution is I just got it delayed like 

`
tmp_2fa = totp.now()
tmp_fetch_open_orders = trading_client.fetchOpenOrders(symbol,params={'twofa':totp.now()})
while totp.now()==tmp_2fa: continue
print(str(trading_client.create_limit_sell_order(symbol,amount,float(price)+price_interval,{'twofa':totp.now()})))`

Thank you so much to read my problem.
